recipes-tag/23-id-1-csx1-analysis/meta.yaml

```diff
@@ -1,14 +1,14 @@
-{% set year = "2018" %}
-{% set cycle = "2" %}
+{% set year = "2019" %}
+{% set cycle = "1" %}
 {% set version ="0" %}
 
 package:
   name: 23-id-1-csx1-analysis
   version: {{ year }}C{{ cycle }}.{{ version }}
 
 build:
-  number: 1
+  number: 0
 
 requirements:
   run:
-    - csxtools
+    - csxtools 0.1.13
```

Comment:
Do we need to pin it to _exactly_ this version or only at least this version?

Reply:
I prefer to use stable version 0.1.13 at this moment. I remember I saw bugs in 0.1.14, but didn't get time to follow up. I may try to update to 0.1.14 during mid-cycle after I am sure 0.1.14 is safe to go.